templates/strings/objc-m.stencil

```diff
@@ -0,0 +1,81 @@
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+{% if tables.count > 0 %}
+#import "{{ param.headerName|default:"Localizable" }}.h"
+
+@interface BundleToken : NSObject
+@end
+
+@implementation BundleToken
+@end
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wformat-security"
+
+static NSString* tr(NSString *tableName, NSString *key, ...) {
+    va_list args;
+    va_start(args, key);
+
+    NSBundle *bundle = [NSBundle bundleForClass:BundleToken.class];
+    NSString *format = [bundle localizedStringForKey:key value:@"" table:tableName];
+    NSString *result = [[NSString alloc] initWithFormat:format arguments:args];
```

Comment:
`va_end(args);` here?

Reply:
Right. And maybe move the `va_start` & stuff to right before this, to make it clear what it's for.

Reply:
I'll do it!